pandas/tests/typing/valid/test_series.py

```diff
@@ -0,0 +1,590 @@
+"""
+Copyright (c) Virtus Lab sp. z o.o. (Ltd.)
+
+Distributed under the terms of the MIT license.
+
+The full license is in the STUBS_LICENSE file, distributed with this software.
+"""
+# flake8: noqa: F841
+# TODO: many functions need return types annotations for pyright
+# to run with reportGeneralTypeIssues = true
+
+import tempfile
```

Comment:
When I run isort (5.9.3) locally on your files, isort makes the same changes as the CI. black (21.9b0) doesn't make any changes (neither before or after running isort).

Reply:
It might help if you'll share the command you're using locally. I've tried `isort pandas/tests/typing/` and `pre-commit run --all-files isort` (version 5.10.1). They both modify the files but hooks modify them back to the previous state.

I've also tried applying the change from the CI and the same situation happens.

I must assume that my local configuration is invalid. I'll try to apply the changes again and push without the hooks.

Reply:
seems to have worked :) I just used `isort pandas/tests/typing/valid/*.py`